app/views/govuk_component/analytics_meta_tags.raw.html.erb

```diff
@@ -0,0 +1,15 @@
+<%
+  organisations = []
+  organisations += links_hash["organisations"] || []
```

Comment:
Given that we know we want to render meta tags from attributes that aren't in the links hash, should this instead take a content item hash?